src/components/MaterialUI/Inputs/InputBase.js

```diff
@@ -141,6 +142,7 @@ const InputBase = ({ inputProps }) => {
 					endAdornment={endAdornment}
 					rows={4}
 					title={value}
+					autoComplete={autoComplete === "off" ? "new-password" : autoComplete}
```

Comment:
So, there would be not way to turn off the "autoComplete" if someone wants to one day. Not sure it is a good idea.

Reply:
If autoComplete is null (the default) the default autoComplete behaviour is in place.  I tested this to make sure nothing changes unless you want to change it.  You can also set the autoComplete to the field that it will pull from, which is kind-of cool.